sakura_core/view/CEditView.cpp

```diff
@@ -708,11 +708,10 @@ LRESULT CEditView::DispatchEvent(
 //		MYTRACE( L"	WM_VSCROLL nPos=%d\n", GetScrollPos( m_hwndVScrollBar, SB_CTL ) );
 		//	Sep. 11, 2004 genta 同期スクロールの関数化
 		{
-			CLayoutInt Scroll = OnVScroll(
-				(int) LOWORD( wParam ), ((int) HIWORD( wParam )) * m_nVScrollRate );
-
 			//	シフトキーが押されていないときだけ同期スクロール
 			if(!GetKeyState_Shift()){
+				CLayoutInt Scroll = OnVScroll(
+					(int)LOWORD(wParam), ((int)HIWORD(wParam)) * m_nVScrollRate);
```

Comment:
```suggestion
					LOWORD(wParam), HIWORD(wParam) * m_nVScrollRate);
```

Reply:
@berryzplus fix it https://github.com/sakura-editor/sakura/pull/1860/commits/1fc1801a340c0d1ebb2b306d27b3d48c14ec2c17